pkg/database/migrations/7_performer_optimization.up.sql

```diff
@@ -0,0 +1,102 @@
+DROP INDEX `performers_checksum_unique`;
+DROP INDEX `index_performers_on_name`;
+DROP INDEX `index_performers_on_checksum`;
+ALTER TABLE `performers` RENAME TO `temp_old_performers`;
+CREATE TABLE `performers` (
+  `id` integer not null primary key autoincrement,
+  `checksum` varchar(255) not null,
+  `name` varchar(255),
+  `gender` varchar(20),
+  `url` varchar(255),
+  `twitter` varchar(255),
+  `instagram` varchar(255),
+  `birthdate` date,
+  `ethnicity` varchar(255),
+  `country` varchar(255),
+  `eye_color` varchar(255),
+  `height` varchar(255),
+  `measurements` varchar(255),
+  `fake_tits` varchar(255),
+  `career_length` varchar(255),
+  `tattoos` varchar(255),
+  `piercings` varchar(255),
+  `aliases` varchar(255),
+  `favorite` boolean not null default '0',
+  `created_at` datetime not null,
+  `updated_at` datetime not null,
+  `image` blob not null
+);
+CREATE UNIQUE INDEX `performers_checksum_unique` on `performers` (`checksum`);
+CREATE INDEX `index_performers_on_name` on `performers` (`name`);
+CREATE INDEX `index_performers_on_checksum` on `performers` (`checksum`);
```

Comment:
There's a redundant index on checksum here (it was in the original schema). I think we can safely remove the non-unique index.

Reply:
I just checked also. _index_performers_on_checksum_ is used now for queries but when dropped _performers_checksum_unique_ gets used instead. So safe to remove.

Reply:
Good point, I've removed `index_performers_on_checksum`.